src/assets/scss/_global.scss

```diff
@@ -1,3 +1,10 @@
+@font-face {
+  font-family: 'gibsonlight';
```

Comment:
Good spot, worked locally but just re-checked and noticed they don't in the email clients. Will see if it's possible another way